wb_schema.gql

```diff
@@ -566,6 +566,7 @@ type Artifact {
   currentManifest: ArtifactManifest
   historyStep: Int64
   ttlDurationSeconds: Int64!
+  tags: [Tag!]!
```

Comment:
note: added to reflect the `Artifact.tags` field added to the graphql schema in: https://github.com/wandb/core/pull/22626